source/install/desktop-app-install.rst

```diff
@@ -18,138 +18,187 @@ Desktop application install guides
+    Unofficial, community-driven ``.rpm`` packages are available.
 
-2. From the **\Downloads** folder, right-click on the file ``mattermost-desktop-setup-5.0.4-win.exe`` and select **Open** to start an installer for the app. Once finished, the Mattermost desktop app will open automatically.
+    1. Download the latest version of the Mattermost Desktop App:
 
-MSI Installer and Group Policies (Beta)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+      - 32-bit systems `mattermost-desktop-5.0.4-linux-i686.rpm <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-i686.rpm>`__
+      - 64-bit systems `mattermost-desktop-5.0.4-linux-x86_64.rpm <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-x86_64.rpm>`__
 
-Download the latest version of the Mattermost desktop app MSI installer (Beta):
+    2. At the command line, execute one of the following commands based on the package you downloaded:
 
-- MSI for `64-bit version of Windows <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-x64.msi>`__
-- MSI for `32-bit version of Windows <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-x86.msi>`__
+      - 32-bit systems ``sudo rpm -i mattermost-desktop-5.0.4-linux-i686.rpm``
+      - 64-bit systems ``sudo rpm -i mattermost-desktop-5.0.4-linux-x86_64.rpm``
 
-`See here <https://docs.mattermost.com/install/desktop-msi-gpo.html>`__ for instructions on installing the Mattermost desktop app via an MSI installer and configuring supported Group Policies. Changes to Group Policies require Mattermost to be restarted for those changes to take effect.
+    3. Run Mattermost as a desktop app.
 
-The following Group Policies are available:
+    **Arch Linux-based systems**
 
-+----------------------------+-----------------------------------------------------------------------------+----------------------+
-| Group Policy               | Description                                                                 | Required Version     |
-+============================+=============================================================================+======================+
-| Enable Server Management   | If disabled, management of servers in the app settings are disabled.        | 4.3 or later         |
-+----------------------------+-----------------------------------------------------------------------------+----------------------+
-| Default Server List        | Define one or more default, permanent servers.                              | 4.3 or later         |
-+----------------------------+-----------------------------------------------------------------------------+----------------------+
+    To install the Desktop client on Arch Linux, see the `Mattermost page <https://wiki.archlinux.org/index.php/Mattermost>`__ on the Arch Linux wiki.
 
-macOS 10.9+
------------
+    **Snapcraft package**
```

Comment:
Do we want to continue documenting a snap option in the product documentation? If so, need updated download URLs